utils/config.go

```diff
@@ -220,8 +227,7 @@ func configureLog(s LogSettings) {
 // then ../config/fileName and last it will look at fileName
 func LoadConfig(fileName string) {
 
-	fileName = findConfigFile(fileName)
-	l4g.Info("Loading config file at " + fileName)
```

Comment:
Can we just move it to after the logging is configured? How about "Loaded config file at..."